.github/workflows/test.docs.yml

```diff
@@ -15,8 +15,8 @@ jobs:
         with:
           filters: |
             docs:
-              - '.github/workflows/**/*'
```

Comment:
The filter `.github/workflows/**/*` is a learning we had in the past.

If you change the workflow, it doe snot run anymore. But the next time the filter applies again, it will run and can fail.
To prevent this, we learned that every workflow should also match the workflows at all times.

While many of your changes still include `**/*` which would match the workflows this particular change does not include the workflows anymore.